File: leetcode/easy/3731-find-missing-elements/solution.java
class Solution {

    public List<Integer> findMissingElements(int[] nums) {

        List<Integer> ans = new ArrayList<>();

        int mannu = nums[0];
        int mixu = nums[0];

        for(int i=1;i<nums.length;i++){

            if(nums[i]<mannu)
                mannu=nums[i];

            if(nums[i]>mixu)
                mixu=nums[i];
        }

        for(int num=mannu;num<=mixu;num++){

            boolean found=false;

            for(int j=0;j<nums.length;j++){

                if(nums[j]==num){

                    found=true;
                    break;
                }
            }

            if(!found)
                ans.add(num);
        }

        return ans;
    }
}

File: leetcode/easy/3731-find-missing-elements/solution.cpp
class Solution {
public:
    vector<int> findMissingElements(vector<int>& nums) {
        vector<int> ans; // empty array ( vector ans)

        int mn = *min_element(nums.begin(), nums.end());   //min
        int mx = *max_element(nums.begin(), nums.end());   // max

        unordered_set<int> st(nums.begin(), nums.end());

        for (int i = mn; i <= mx; i++) {
            if (st.find(i) == st.end()) {
                ans.push_back(i);
            }
        }

        return ans;
    }
};